modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -188,3 +188,4 @@ export {
 	IInternalRenderer,
 } from './FrontendDataSetContext';
 export {INTERNAL_CELL_RENDERERS as FDS_INTERNAL_CELL_RENDERERS} from './cell_renderers/InternalCellRenderer';
+export {NESTING_DELIMITER, NESTING_ROOT} from './Constants';
```

Comment:
We started prefixing exports from FDS with `FDS_`, like `FDS_EVENT`. Locally, it can be `EVENT`, there is no need for prefix. I think it makes the code a bit a more readable in the end module. Also, we should probably be a bit more descriptive with `NESTING_*`, is is clear in local context but not so much in global. 

Maybe something like `(FDS_)NESTED_FIELD_NAME_DELIMITER` and `(FDS_)NESTED_FIELD_NAME_PARENT_SUFFIX`?